README.md

```diff
@@ -13,7 +13,9 @@ A data source for a `UITableView` or a `UICollectionView` can be constructed by
 
 ## Installation
 
-The recommended way to add DataSource to your project is with [Carthage](https://github.com/Carthage/Carthage/releases/).
+The recommended way to add DataSource to your project is with CocoaPods. Add `pod 'DataSource', :git => 'https://github.com/Fueled/DataSource.git'` to your Podfile and run `pod install`.
```

Comment:
I would rather us stay impartial and just include 2 subsections: `### Cocoapods` and `### Carthage` (in this order). The reason being, Carthage was recommended as an alternative to no dependency management. What do you think?

Reply:
Sebsections are a common practice and make it easier to find relevant information (compared to reading text paragraphs).
If you don't want to be impartial you can write `### CocoaPods (recommended)` but I don't see a reason for doing that.